src/components/Common/AvatarEditModal.tsx

```diff
@@ -231,20 +292,45 @@ const AvatarEditModal = ({
         </DialogHeader>
         <div className="flex h-full w-full items-center justify-center overflow-y-auto">
           <div className="flex max-h-screen min-h-96 w-full flex-col overflow-auto">
+            {errorMessage && (
+              <div className="mb-4 p-3 bg-red-50 border border-red-200 rounded-md text-red-600">
+                <p className="flex items-center">
+                  <CareIcon icon="l-exclamation-circle" className="mr-2" />
+                  {errorMessage}
+                </p>
+              </div>
+            )}
             {!isCameraOpen ? (
               <>
                 {preview || imageUrl ? (
                   <>
-                    <div className="flex h-[50vh] md:h-[75vh] w-full items-center justify-center overflow-scroll rounded-lg border border-secondary-200">
-                      <img
-                        src={
-                          preview && preview.startsWith("blob:")
-                            ? DOMPurify.sanitize(preview)
-                            : imageUrl
-                        }
-                        alt="cover-photo"
-                        className="h-full w-full object-contain"
-                      />
+                    <div className="flex flex-1 items-center justify-center rounded-lg relative">
+                      <div
+                        className="w-full"
+                        style={{
+                          overflow: "hidden",
+                          maxWidth: "100%",
+                          maxHeight: "calc(100vh - 200px)",
+                        }}
+                      >
+                        {isCropping ? (
+                          <Cropper
+                            image={preview || imageUrl || undefined}
+                            crop={crop}
+                            zoom={zoom}
+                            aspect={1}
+                            onCropChange={setCrop}
+                            onZoomChange={setZoom}
+                            onCropComplete={onCropComplete}
+                          />
+                        ) : (
+                          <img
+                            src={preview || imageUrl}
```

Comment:
Sanitize, add the code with DOMPurify back in. You can do the same for the other warning above in getCroppedImg (in createImg function).